### DISCUSSION TITLE
Context-Sensitive scrollbar with alphabetical / date-based cues

### DISCUSSION THREAD

**Discussion by lagset** [astubenbord/paperless-mobile#192] [Feature Requests]
Hi there,

again great app. I am currently using the beta on my Pixel 3a and is works well. What I observed while using the app: I kept scrubbing the right edge of my display while searching for specific correspondents, tags or document types in the label views. Why? I expected a scrollbar to be on the right side and wanted to use it to quickly got a correspondent beginning with a specific letter. 

Android is using this e.g. in the native "contacts" app.

It could look something like this and could also be extended to the default document view, showing the month and year while scrolling.

I don't know, how complex it to implement this feature, but it is something I would definitely use heavily in my workflows.

<img width="1013" alt="paperless-mobile_scrollbar-proposition" src="https://github.com/astubenbord/paperless-mobile/assets/15156011/b0b572b0-61a7-4d03-8ab4-06283fd229f8">

**Comment by astubenbord** [comment]
Hi, thanks for the idea, I think it's really useful! 

I've never implemented something like this in flutter and I assume it's not trivial. I'll put it on the backlog and check it out once I have more time.
